sdks/java/build-tools/src/main/resources/beam/findbugs-filter.xml

```diff
@@ -0,0 +1,479 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+  <Bug category="I18N"/>
+  <Bug pattern="DM_STRING_CTOR"/>
+  <Bug pattern="SE_NO_SERIALVERSIONID"/>
+
+
+  <!--
+          Baseline issues below. No new issues should be added to this list. Instead, suppress
+          the issue inline using @SuppressFBWarnings and a documented reason, or (preferably) fix
+          the issue.
+        -->
+  <Match>
+    <Class name="org.apache.beam.sdk.coders.AvroCoder"/>
```

Comment:
Yup. This uncovered tons of things that probably-should-be-fixed-maybe-or-at-least-needs-consideration.  I only tackled a couple here as an example. Eventually we should work through the baseline and convert them all to inline suppressions with linked bugs or justification why we don't need to fix them.